src/openrct2/paint/paint.h

```diff
@@ -123,6 +123,15 @@ typedef struct sprite_bb {
     rct_xyz16 bb_size;
 } sprite_bb;
 
+typedef struct paint_struct_bound_box {
```

Comment:
You could also replace the members of paint_struct with the new struct.

Reply:
Yes, this is coming in future commit.